app/main.py

```diff
@@ -1,25 +1,83 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self._name = ""
+
+    def __set_name__(self, owner: object, name: str) -> None:
+        self._name = "_" + name
+
+    def __get__(
+            self,
+            obj: object,
+            object_type: type | None = None
```

Comment:
invalid signature + typehints for __get__
            self,
            instance: ...,
            owner: ...